resources/js/visual-editor/blocks/_shared/social-icons.css

```
@@ -0,0 +1,150 @@
+.ap-author-social-icons__chip:hover,
+.ap-author-social-icons__chip:focus-visible {
+    color: var(--ap-social-hover-color, var(--ap-social-color, currentColor));
+    background-color: var(--ap-social-hover-bg, var(--ap-social-bg, transparent));
+}
+
+.ap-social-share-content__icon,
+.ap-author-social-icons__icon {
+    width: 1em;
+    height: 1em;
+    flex-shrink: 0;
+    fill: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)